src/worker_pool.rs

```diff
@@ -432,3 +465,76 @@ fn verify_policy_settings(
         Err(anyhow!("{}", errors.join(", ")))
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use std::env;
+
+    #[test]
+    fn valid_kubewarden_version_test() {
+        let kubewarden_version = Version::parse(env!("CARGO_PKG_VERSION")).ok().unwrap();
```

Comment:
Generally speaking, you don't need to do `ok().unwrap()`, just use `unwrap`.

However, just use the lazy static constant you defined ahead. Also, give we have this constant now, I would change these tests to use [rstest](https://crates.io/crates/rstest) to DRY the code